src/black/linegen.py

```diff
@@ -305,9 +305,9 @@ def visit_STRING(self, leaf: Leaf) -> Iterator[Line]:
             quote_len = 1 if docstring[1] != quote_char else 3
             docstring = docstring[quote_len:-quote_len]
             docstring_started_empty = not docstring
+            indent = " " * 4 * self.current_line.depth
 
             if is_multiline_string(leaf):
-                indent = " " * 4 * self.current_line.depth
```

Comment:
This should most likely also have the preview check, or I've missed some context.

Reply:
It doesn't make any difference to the output, as if it is a multiline string then it's identical, and otherwise indent doesn't get used. I can put it in the preview if you like? Is the idea of the preview to keep all the code the same or unless previewing or just keep the formatting the same?

Reply:
Right, that was the missing context then 😄 It's a-ok! Having all refactors in preview would be.. not good. Just the formatting 👍